### ISSUE TITLE
'.' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by awayken** [danbroooks/gulp-phantomcss#12]
Trying to install this plugin on Windows throws an error. It wasn't throwing an error back on 0.0.3, but I recently tried to upgrade to 0.1.0.

```shell
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "D:\\Program Files\\nodejs\\\\node.exe" "D:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "gulp-phantomcss" "-D"
npm ERR! node v0.12.5
npm ERR! npm  v2.11.2
npm ERR! code ELIFECYCLE

npm ERR! gulp-phantomcss@0.1.0 postinstall: `./node_modules/bower/bin/bower install phantomcss`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gulp-phantomcss@0.1.0 postinstall script './node_modules/bower/bin/bower install phantomcss'.
npm ERR! This is most likely a problem with the gulp-phantomcss package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./node_modules/bower/bin/bower install phantomcss
npm ERR! You can get their info via:
npm ERR!     npm owner ls gulp-phantomcss
npm ERR! There is likely additional logging output above.
```

**Comment by Novak99** [comment]
I get the same error, using  Windows10.

**Comment by danbroooks** [comment]
Could you guys try installing 0.1.1 ? 

Thanks

**Comment by Novak99** [comment]
Version of what?

npm view gulp-phantomcss versions
[ '0.0.1', '0.0.2', '0.0.3', '0.0.4', '0.1.0' ]

npm install gulp-phantomcss@0.1.1
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program
Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
"gulp-phantomcss@0.1.1"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3

npm ERR! version not found: gulp-phantomcss@0.1.1

On Wed, Aug 12, 2015 at 10:22 PM, Dan Brooks <notifications@github.com>
wrote:

> Could you guys try installing 0.1.1 ?
>
> Thanks
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/danbroooks/gulp-phantomcss/issues/12#issuecomment-130435479>
> .
>



-- 
S poštovanjem,
Novak Lukić

**Comment by awayken** [comment]
@danbroooks I was able to install 0.1.1. When I then tried to run my gulp task, I got an error. When I get a chance, I will provide you with more details.

**Comment by awayken** [comment]
@danbroooks: I've recently taken another look at this. I installed 0.1.1, and which installs correctly. This is the error I get when I try to run my task:

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn D:\Sites\acx-signon\node_modules\gulp-phantomcss\node_modules\.bin\phantomjs ENOENT
    at exports._errnoException (util.js:870:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickCallback (node.js:356:17)
```

node -v: v4.4.1
npm -v: 3.7.2
gulp -v: [09:27:20] CLI version 3.9.1
[09:27:20] Local version 3.9.1
phantomjs -v: 1.9.8

Let me know if I can provide any other information.

**Comment by awayken** [comment]
This issue appears to be fixed in 0.2.0. I'm having issues with 0.2.0 because of changes in the PhantomJS that it's using, but this is not the place for those issues. I guess this ticket is more or less resolved, then.
